### ISSUE TITLE
python 3.7 incompatibility

### ISSUE THREAD

**Issue by josephbiko** [mosquito/aio-pika#132]
```
async for message in queue:
        with message.process():
            print(message.body)

            if queue.name in message.body.decode():
                break
```

throws an error:
> 'async for' received an object from __aiter__ that does not implement __anext__: generator

**Comment by hartym** [comment]
Also caused by the dependency to "pika==0.10.0", which is not compatible with python3.7, because of "async" being now a keyword.

**Comment by decaz** [comment]
@mosquito is it possible to adapt `aio-pika` to `pika==0.12.0` for Python 3.7 compatibility?

**Comment by josephbiko** [comment]
In the mean time:
`  for msg in q:
                func(await msg)`

this works as replacement

**Comment by mosquito** [comment]
I released the `aio-pika-3.0.0`. Now `pika` vendored to aio-pika. It solution is a part of my plan to creating a my own amqp protocol driver.

**Comment by lukebakken** [comment]
I have worked on Pika / `aio-pika` compatibility, and, as you can see, I had planned on ensuring that Pika and `aio-pika` are compatible. Is there something that we could have done to keep the two projects working together?

Most importantly, please ensure that what you have done follows Pika's [`LICENSE`](https://github.com/pika/pika/blob/f5eca961588277e2c26d818d5037119901781fcc/LICENSE) requirements. You are now re-distributing Pika source code and must include Pika's `LICENSE` file in your distribution, which you appear to not be doing. Please change that as soon as possible.
